plugins/woocommerce-admin/client/layout/store-alerts/index.js

```diff
@@ -218,63 +224,66 @@ export class StoreAlerts extends Component {
 					>
 						{ alert.title }
 					</Text>
-					{ numberOfAlerts > 1 && (
-						<div className="woocommerce-store-alerts__pagination">
-							<Button
-								onClick={ this.previousAlert }
-								disabled={ currentIndex === 0 }
-								label={ __( 'Previous Alert', 'woocommerce' ) }
-							>
-								<Icon
-									icon={ chevronLeft }
-									className="arrow-left-icon"
-								/>
-							</Button>
-							<span
-								className="woocommerce-store-alerts__pagination-label"
-								role="status"
-								aria-live="polite"
-							>
-								{ interpolateComponents( {
-									mixedString: __(
-										'{{current /}} of {{total /}}',
+					<div>
```

Comment:
Why was this extra `div` needed?

Reply:
This is needed to group the pagination buttons with the close buttons, since they're inside a div with display: flex

<img width="1628" alt="image" src="https://user-images.githubusercontent.com/13437655/234591664-41a878ed-0717-4b56-aac6-3b5256624d75.png">